docs/audit.md

```diff
@@ -197,24 +197,24 @@ detect-secrets audit --report --fail-on-unaudited --fail-on-live --fail-on-audit
 
 Below are four documented methods for adding detect-secrets reporting to your pipeline.
 
-**It is recommended to [use the `detect-secrets-redhat-ubi` Docker image](#using-detect-secrets-redhat-ubi-docker-image)**.
+**It is recommended to [use the `detect-secrets:redhat-ubi` Docker image](#using-detect-secrets:redhat-ubi-docker-image)**.
 
-#### Using `detect-secrets-redhat-ubi` Docker Image
+#### Using `detect-secrets:redhat-ubi` Docker Image
 
 This Docker image offers additional benefits over the general-purpose [`detect-secrets` Docker image](#using-detect-secrets-docker-image). One being additional security, and the other is that the Red Hat Universal Base Image is [OCI-compliant](https://opencontainers.org/faq/).
 
 To use this image in your pipeline, add the following commands to your pipeline script:
 
 1. Get the latest image:
-    - `docker pull ibmcom/detect-secrets-redhat-ubi:latest`
+    - `docker pull ibmcom/detect-secrets:redhat-ubi:latest`
```

Comment:
```suggestion
    - `docker pull ibmcom/detect-secrets:redhat-ubi`
 ```
<img width="758" alt="Screen Shot 2022-03-30 at 10 26 22 AM" src="https://user-images.githubusercontent.com/43279515/160858487-edd6a148-d183-4d7a-9e5a-854707aa4d33.png">

You cannot have 2 tags (redhat-ubi and latest). To fix this, going forward we will need to have a few different ways of versioning this specific image 

- ibmcom/detect-secrets:redhat-ubi
    - This tag will be considered as the "latest" version
- ibmcom/detect-secrets:0.13.1-ibm.48.dss-redhat-ubi
    - This one will allow users to peg themselves to a specific version of the image if they want to. Just incase they dont want the latest image 

This versioning technique is similar to how node version's their docker images https://hub.docker.com/_/node